docs/docs/cmd/tenant/auditlog/auditlog-report.md

```diff
@@ -0,0 +1,46 @@
+# tenant auditlog report
+
+Gets audit logs from the Office 365 Management API
+
+## Usage
+
+```sh
+m365 tenant auditlog report [options]
+```
+
+## Options
+
+`-c, --contentType <contentType>`
+: Audit content type of logs to be retrieved, should be one of the following: AzureActiveDirectory, Exchange, SharePoint, General, DLP.
+
+`-s, --startTime [startTime]`
+: Start time of logs to be retrieved. Start time and end time must both be specified (or both omitted) and must be less than or equal to 24 hours apart.
+
+`-e, --endTime [endTime]`
+: End time of logs to be retrieved. Start time and end time must both be specified (or both omitted) and must be less than or equal to 24 hours apart.
+
+--8<-- "docs/cmd/_global.md"
+
+## Remarks
+
+By default, if startTime and endTime are not mentioned, then the content available in the last **24 hours** is returned. `StartTime` and `EndTime` must be less than or equal to **24 hours** apart, with the `StartTime` prior to `EndTime` and `StartTime` no more than 7 days in the past.
+
+`DLP` audit log data is only available to users that have been granted “Read DLP sensitive data” permissions. Otherwise you will get `Error: Request failed with status code 401`
+
+## Examples
+
+Gets audit logs from the Office 365 Management API for the `Exchange` ContentType.
+
+```sh
+m365 tenant auditlog report --contentType "Exchange"
+```
+
+Gets audit logs from the Office 365 Management API for the `Exchange` ContentType in the date range between `2020-12-13` and `2020-12-14`
+
+```sh
+m365 tenant auditlog report --contentType "Exchange" --startTime "2020-12-13" --endTime "2020-12-14"
+```
```

Comment:
Would it be possible to add some examples on how to search within a day (between specific hours)?

Reply:
Sure. I will add the examples to show the usage of the command for a certain time period, especially time period within a day.